pkg/beyla/config_compat.go

```diff
@@ -15,3 +19,18 @@ type internalPromConfig struct {
 	Port int    `yaml:"port,omitempty" env:"BEYLA_INTERNAL_METRICS_PROMETHEUS_PORT"`
 	Path string `yaml:"path,omitempty" env:"BEYLA_INTERNAL_METRICS_PROMETHEUS_PATH"`
 }
+
+type nameResolverConfig struct {
```

Comment:
Can we not use the OBI version of this config? We are able to replace the env variable names, so I think it should be possible to remove this too and use the OBI version.